src/components/Navbar/NavTabs.js

```diff
@@ -33,64 +34,74 @@ import "./NavAccessibility.css"
       return () => window.removeEventListener("resize", handleResize);
     }, []); // Empty array ensures that effect is only run on mount
 
-    // The padding value as a function of screen size
-    let navbarPadding,navbarHeight, navitemWidth, logoWidth, logoFontSize; 
-    let buttonHeight, buttonWidth, heartFontSize, heartWidth;
-    let navtabsDisplay, burgerMenuDisplay;
+    // Navbar and childs slyle properties
+    let navbarSideMargin, navbarHeight, navtabsDisplay,navitemWidth;
+    let logoWidth, logoFontSize, logoTextMaxWidth; 
+    let buttonHeight, buttonWidth, buttonFontSize, heartWidth;
+    let burgerMenuDisplay, burMenuButtonPosition;
     
     if (windowSize.width >= 1040) {
-      navbarPadding = (windowSize.width / 4 - 240).toFixed(1);
+      navbarSideMargin = (windowSize.width / 4 - 240).toFixed(1);
       navbarHeight = 80;
       navitemWidth = ((windowSize.width - 720) / 5.5).toFixed(1);
       logoFontSize = 20;
       logoWidth = 48;
+      logoTextMaxWidth = 250;
       buttonHeight = 48;
       buttonWidth = 156;
       navtabsDisplay = "inline-block"
       burgerMenuDisplay = "none"
-      heartFontSize = 18;
+      burMenuButtonPosition = "55%"
```

Comment:
isnt burgerMenu not visible in this media query, then do we need this burMenuButtonPosition ?

Reply:
@ we don't need in real. Can be removed.

Reply:
@BBoutet1  once u make this change, feel free to resolve this conversation